agent/__init__.py

```diff
@@ -711,11 +718,40 @@ def run(ping=True, debug=False, dev=False):
             f.write(crt['crt'])
         os.chmod(COMBINED_PEM_PATH, 0o600)
 
-        print("Writing config...")
+        log_or_print("Writing config...")
         config = configparser.ConfigParser()
         config['DEFAULT'] = {'fallback_token': crt['fallback_token']}
         with open(INI_PATH, 'w') as configfile:
             config.write(configfile)
         os.chmod(INI_PATH, 0o600)
 
-        send_ping(debug=debug, dev=dev)
+        send_ping(dev=dev)
+
+
+def setup_logging(
+    path=LOG_CONFIG_PATH,
+    level=logging.INFO,
+    log_format="%(asctime)s - %(name)s - %(levelname)s - %(message)s",
+):
+    """
+    Setup logging configuration
+    - if found path, use it as log config ini file
+    - else use basicconfig() with level and log_format as parameter
+    - if there is `log_level` item in wott-agent `config.ini`, it tries to be used as actual log level
+    """
+    if os.path.exists(path):
+        logging.config.fileConfig(path)
+    else:
+        logging.basicConfig(level=level, format=log_format)
+
+    log_level = get_ini_log_level()
```

Comment:
Why do you take this log lever from the primary config if it's never used?

Reply:
fixed